standard/docs/en/implementation/milestones.md

```diff
@@ -0,0 +1,52 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation blocks. 
```

Comment:
The Getting Started guidance refers to planning, tender, contract and contract implementation as 'sections' (as distinct from building block fields that include milestones) so maybe have the last word as 'sections' to be consistent.